Voycar.Api.Web.Tests.Integration/Setup/ClientFactory.cs

```diff
@@ -21,9 +22,15 @@ public static async Task<HttpClient> CreateMemberClient(AppFixture<Program> app,
     {
         const string testMail = "member.integration@test.de";
         const string password = "integration";
+        const string planName = "basic";
 
         var member = app.CreateClient();
-
+        var planId = (await context.Plans.FirstOrDefaultAsync(
+            plan => plan.Name == planName))?.Id;
+        if (planId is null)
+        {
+            throw new RowNotInTableException($"plan \"{planId}\" is not in db");
```

Comment:
Da sollte probably der `planName` rein
```suggestion
            throw new RowNotInTableException($"plan \"{planName}\" is not in db");
```

Reply:
oh true